### PR TITLE
Bug 1241535 - Add support to job actions for collecting gecko profile…

### PR DESCRIPTION
**Pull request by jmaher**
…s of performance tests.

### CODE REVIEW

File: ui/job-view/GeckoProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import React from 'react';
```
Comment (edmorley) [inline ui/job-view/GeckoProfile.jsx]: I think this file is unused? (I presume leftover from before the in-tree task was created?)

Comment (jmaher) [reply ui/job-view/GeckoProfile.jsx]: thanks, I had originally had that file and forgot to 'git rm'; apologies for sloppy pushes

Comment (edmorley) [reply ui/job-view/GeckoProfile.jsx]: Not a problem at all! Just though it would give Cameron less to review :-)

**Comment by edmorley** [comment]
Adding @helfi92 to also have a look at this, since he wrote many of the other tasks :-)

**Comment by jmaher** [comment]
@camd I updated this to have a better error message for the end user when the in-tree taskcluster code isn't present, please take a quick peek

File: ui/job-view/details/summary/ActionBar.jsx [hunk 2]
```diff
@@ -64,6 +65,44 @@ class ActionBar extends React.Component {
...
+      return this.thNotify.send('Must be logged in to create a gecko profile', 'danger');
+    }
+
+    getGeckoDecisionTaskId(
+      selectedJob.push_id).then(decisionTaskId => (
+      TaskclusterModel.load(decisionTaskId, selectedJob).then((results) => {
+        const geckoprofile = results.actions.find(result => result.name === 'geckoprofile');
+
+        if (geckoprofile === undefined || !geckoprofile.hasOwnProperty('kind')) {
+          return this.thNotify.send('Job was scheduled without taskcluster support for GeckoProfiles');
```
Comment (camd) [inline ui/job-view/details/summary/ActionBar.jsx]: Looks great!  :)

**Comment by camd** [comment]
@jmaher : Your new message looks great, thanks!  :)

**Pull request merged**